docs/connectors/sources/debezium.md

```diff
@@ -0,0 +1,226 @@
+# Debezium input connector
+
+:::note
+This page describes configuration options specific to the Debezium source connector.
+See [top-level connector documentation](/docs/connectors/) for general information
+about configuring input and output connectors.
+:::
+
+Debezium is a widely-used **Change Data Capture** (CDC) technology that streams real-time changes from databases such
+as PostgreSQL, MySQL, and Oracle to Kafka topics. Feldera can consume these change streams as inputs. We support
+Debezium streams encoded in both [JSON](/docs/formats/json.md) and [Avro](/docs/formats/avro.md) formats. Synchronizing
```

Comment:
btw maybe we can have  a table in the higher level connectors page that shows which connector supports which format